.github/workflows/projects.yml

```diff
@@ -0,0 +1,20 @@
+name: Auto Assign to Project(s)
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Add to Kanban Board
+    steps:
+    - name: Add NEW issues and pull requests to kanban board
+      uses: srggrs/assign-one-project-github-action@1.2.1
+      if: github.event.action == 'opened'
```

Comment:
Is this ever not true, with the trigger configuration above?

Reply:
no. I left it there for future-proofing, but I guess it's unlikely that we'll make things more complicated than a single kanban board.